===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(do_mkfs_exec_wrapper:mkfs)) at 3 (line 763, column 2) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/lxc/src/lxc/bdev/bdev.c
+++ b/table1/benchmarks/lxc/src/lxc/bdev/bdev.c
@@ -760,6 +760,7 @@ int do_mkfs_exec_wrapper(void *args)
 	TRACE("executing \"%s %s\"", mkfs, data[1]);
 	execlp(mkfs, mkfs, data[1], (char *)NULL);
 	SYSERROR("failed to run \"%s %s \"", mkfs, data[1]);
+  free(mkfs);
 	return -1;
 }
 
